coins/ethereum/src/ethereum_wallet.rs

```diff
@@ -340,6 +492,36 @@ impl EthereumWallet {
         self.public_address.clone()
     }
 
+    /// A convenience method for retrieving the string of a public_address
+    pub fn address(&self) -> String {
+        self.public_address()
+    }
+
+    /// Return the pub/priv keys at a specified index
+    /// For now, we're assuming that the path we're using for derivation is the default (m/44'/60'/0'/0/{index})
+    pub fn index(&self, _index: u64) -> (String, String) {
```

Comment:
what is this function for?

Reply:
I was planning on using it to temporarily derive additional Eth key-pairs, since presently functionality to do so exists in only WalletCLI, unless I'm mistaken.  private and contained

Reply:
It should be something we could retrieve from the key manager